docs/src/modules/javascript/pages/access-control.adoc

```diff
@@ -0,0 +1,42 @@
+= Access Control Lists (ACLs)
+
+This section describes functionality in the JavaScript/TypeScript SDK related to ACLs (Access Control Lists), for information about how the ACLs are defined and control access see https://docs.kalix.io/services/using-acls.html[Using ACLs]
+
+== Default ACL in project templates
+
+If no ACLs is defined at all in a Kalix service Kalix will allow requests from both other services and the internet to all components of a Kalix service.
+
+The Kalix quickstarts include a less permissive ACL for the entire service, to not accidentally make services available to the public internet, just like the one described in the next section.
+
+== Defining an ACL for the entire Kalix Service
+
+A default ACL for the entire Kalix Service can be defined by placing a `kalix_policy.proto` file among the protobuf descriptors of the service, and including it in your build. It should only contain a `kalix.file(acl)` annotation:
```

Comment:
Should that be `(kalix.file).acl` annotation, rather than `kalix.file(acl)`?

```suggestion
A default ACL for the entire Kalix Service can be defined by placing a `kalix_policy.proto` file among the protobuf descriptors of the service, and including it in your build. It should only contain a `(kalix.file).acl` annotation:
```